### PR TITLE
Bay station and deform removal.

### PR DESCRIPTION
**Pull request by Arctrooper8**
## About The Pull Request

Deletes a ton of unused stuff from old bay races, most of them didn't even work. Also removes the deformed limb code considering it was completely unused except for a slight line of text you'd occasionally see.

## Why It's Good For The Game

Most of this stuff barely even works, and it's not as though they're ever used at all in the first place.

## Changelog
:cl:
del: Unused baystation races and their codes it gone.
del: Unused deformed limb code is gone.
/:cl:

### CODE REVIEW

File: code/__DEFINES/equipment.dm [hunk 1]
```diff
@@ -86,7 +86,6 @@
 #define HIDELOWHAIR (1<<7)		// temporarily removes the user's facial hair overlay.
 #define HIDETOPHAIR (1<<8)		// temporarily removes the user's hair overlay. Leaves facial hair.
 #define HIDEALLHAIR (1<<9)		// temporarily removes the user's hair, facial and otherwise.
-#define HIDETAIL (1<<10)
 #define HIDEFACE (1<<11)	//Dictates whether we appear as unknown.
```
Comment (TiviPlus) [inline code/__DEFINES/equipment.dm:89]: you need to edit the flags

File: code/__DEFINES/is_helpers.dm [hunk 2]
```diff
@@ -59,13 +52,8 @@
 #define ishumanbasic(H) (is_species(H, /datum/species/human))
 #define iszombie(H) (is_species(H, /datum/species/zombie))
 
-//Monkey species and subtypes
+//Monkey
 #define ismonkey(H) (is_species(H, /datum/species/monkey))
-#define isfarwa(H) (is_species(H, /datum/species/monkey/farwa))
-#define isstok(H) (is_species(H, /datum/species/monkey/stok))
-#define isnaera(H) (is_species(H, /datum/species/monkey/naera))
-#define isyiren(H) (is_species(H, /datum/species/monkey/yiren))
```
Comment (TiviPlus) [inline code/__DEFINES/is_helpers.dm:67]: Keep the monkey subspecies those are useful for mapping

File: code/datums/jobs/job/skeleton.dm [hunk 3]
```diff
@@ -1,56 +0,0 @@
-/datum/job/skeleton
-	access = ALL_ANTAGONIST_ACCESS
-	minimal_access = ALL_ANTAGONIST_ACCESS
-	skills_type = /datum/skills/skeleton
```
Comment (TiviPlus) [inline code/datums/jobs/job/skeleton.dm:4]: This is not related to bay, we added this

File: code/game/objects/items.dm [hunk 4]
```diff
@@ -83,14 +83,7 @@
...
 
-	/* Species-specific sprites, concept stolen from Paradise//vg/.
-	ex:
-	sprite_sheets = list(
-		"Tajara" = 'icons/cat/are/bad'
-		)
-	If index term exists and icon_override is not set, this sprite sheet will be used.
-	*/
-
+	/// Species-specific sprites, concept stolen from Paradise//vg/.
```
Comment (TiviPlus) [inline code/game/objects/items.dm:86]: Why

File: code/game/objects/items/reagent_containers/food/snacks.dm [hunk 5]
```diff
@@ -1365,33 +1365,6 @@
 	icon_state = "monkeycubewrap"
 	package = TRUE
 
-
-/obj/item/reagent_containers/food/snacks/monkeycube/farwacube
-	name = "farwa cube"
-	monkey_type = /mob/living/carbon/human/species/monkey/farwa
-
-/obj/item/reagent_containers/food/snacks/monkeycube/wrapped/farwacube
-	name = "farwa cube"
```
Comment (TiviPlus) [inline code/game/objects/items/reagent_containers/food/snacks.dm:1374]: see comment about monkeys

File: code/game/objects/items/storage/boxes.dm [hunk 6]
```diff
@@ -163,21 +163,6 @@
 	spawn_type = /obj/item/reagent_containers/food/snacks/monkeycube/wrapped
 	spawn_number = 5
 
-/obj/item/storage/box/monkeycubes/farwacubes
-	name = "farwa cube box"
-	desc = "Drymate brand farwa cubes, shipped from Ahdomai. Just add water!"
-	spawn_type = /obj/item/reagent_containers/food/snacks/monkeycube/wrapped/farwacube
```
Comment (TiviPlus) [inline code/game/objects/items/storage/boxes.dm:169]: monjkey

File: code/game/objects/structures/crates_lockers/closets/wardrobe.dm [hunk 7]
```diff
@@ -99,8 +99,6 @@
 	icon_closed = "green"
 
 /obj/structure/closet/wardrobe/xenos/PopulateContents()
-	new /obj/item/clothing/suit/unathi/mantle(src)
-	new /obj/item/clothing/suit/unathi/robe(src)
 	new /obj/item/clothing/shoes/sandal(src)
```
Comment (TiviPlus) [inline code/game/objects/structures/crates_lockers/closets/wardrobe.dm:102]: Why removing clothes)

File: code/modules/mob/living/carbon/human/human_damage.dm [hunk 8]
```diff
@@ -169,38 +169,11 @@
...
-					O.unmutate()
-					to_chat(src, "<span class = 'notice'>Your [O.display_name] is shaped normally again.</span>")
-					return
-
-	if (getCloneLoss() < 1)
-		for (var/datum/limb/O in limbs)
-			if (O.limb_status & LIMB_MUTATED)
-				O.unmutate()
-				to_chat(src, "<span class = 'notice'>Your [O.display_name] is shaped normally again.</span>")
+		..()
```
Comment (TiviPlus) [inline code/modules/mob/living/carbon/human/human_damage.dm:176]: Return ..()? also else is not needed here

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Comment by TiviPlus** [comment]
This really feels like it shoudve been atomized btw

**Comment by Arctrooper8** [comment]
> This really feels like it shoudve been atomized btw

Probably but I got the two things mixed up and it'd be agonizing to untangle it at this point.

File: code/game/objects/items.dm (hunk 4 above)
Comment (Arctrooper8) [reply code/game/objects/items.dm]: Autodoc, I updated it to include an example and the index term info.

File: code/game/objects/structures/crates_lockers/closets/wardrobe.dm (hunk 7 above)
Comment (Arctrooper8) [reply code/game/objects/structures/crates_lockers/closets/wardrobe.dm]: Unused locker and it uses an unathi sprite which does not exist.

File: code/modules/mob/living/carbon/human/human_damage.dm (hunk 8 above)
Comment (Arctrooper8) [reply code/modules/mob/living/carbon/human/human_damage.dm]: Fixed

File: _maps/map_files/LV624/LV624.dmm [hunk 9]
```diff
@@ -329,12 +329,12 @@
...
-	desc = "A curved blade made of a strange material. It looks both old and very sharp.";
-	force = 30;
-	icon_state = "unathiknife";
+/obj/item/weapon/katana{
+	desc = "Woah! Holy shit! A glowing alien sword! How fucking cool is that, and it's super sharp!";
+	force = 85;
+	icon_state = "blade";
 	name = "\improper alien blade";
-	throwforce = 26
+	throwforce = 45
```
Comment (TiviPlus) [inline _maps/map_files/LV624/LV624.dmm:337]: Yea absolutely no remove all these varedits

**Review by TiviPlus — CHANGES_REQUESTED** [review]
lgtm barring the one comemnt

File: _maps/map_files/LV624/LV624.dmm (hunk 9 above)
Comment (Arctrooper8) [reply _maps/map_files/LV624/LV624.dmm:337]: It's probably for the best to remove it (what with varedits being bad), I'll just stick a standard sword of some type in.

**Comment by Derrick2Electricboogaloo** [comment]
So weird I was thinking why do we not delete so many of the old files that never get touched as I was going through code.

**Review by TiviPlus — CHANGES_REQUESTED** [review]
You still havent fixed you removing monkeys
